testv2/tests.yml

```diff
@@ -320,3 +275,159 @@
+    - name: digitalocean_default
+    - name: equinixmetal_default
+    - name: hetzner_default
+    - name: openstack_default
+    - name: vsphere_default
+    - name: vsphere_flatcar
+
+- scenario: install_containerd_external
+  initVersion: v1.22.11
+  infrastructures:
+    - name: aws_amzn
+    - name: aws_centos
+    - name: aws_defaults
+    - name: aws_flatcar
+    - name: aws_rhel
+    - name: aws_rockylinux
+    - name: azure_default
+    - name: digitalocean_default
+    - name: equinixmetal_default
+    - name: hetzner_default
+    - name: openstack_default
+    - name: vsphere_default
+    - name: vsphere_flatcar
+
+- scenario: install_containerd_external
+  initVersion: v1.23.8
+  infrastructures:
+    - name: aws_amzn
+    - name: aws_centos
+    - name: aws_defaults
+    - name: aws_flatcar
+    - name: aws_rhel
+    - name: aws_rockylinux
+    - name: azure_default
+    - name: digitalocean_default
+    - name: equinixmetal_default
+    - name: hetzner_default
+    - name: openstack_default
+    - name: vsphere_default
+    - name: vsphere_flatcar
+
+- scenario: install_containerd_external
+  initVersion: v1.24.2
+  infrastructures:
+    - name: aws_amzn
+    - name: aws_centos
+    - name: aws_defaults
+    - name: aws_flatcar
+    - name: aws_rhel
+    - name: aws_rockylinux
+    - name: azure_default
+    - name: digitalocean_default
+    - name: equinixmetal_default
+    - name: hetzner_default
+    - name: openstack_default
+    - name: vsphere_default
+    - name: vsphere_flatcar
+
+- scenario: install_docker_external
+  initVersion: v1.21.14
+  infrastructures:
+    - name: aws_amzn
+    - name: aws_centos
+    - name: aws_defaults
+    - name: aws_flatcar
+    - name: aws_rhel
+    - name: aws_rockylinux
+    - name: azure_default
+    - name: digitalocean_default
+    - name: equinixmetal_default
+    - name: hetzner_default
+    - name: openstack_default
+    - name: vsphere_default
+    - name: vsphere_flatcar
+
+- scenario: install_docker_external
+  initVersion: v1.22.11
+  infrastructures:
+    - name: aws_amzn
+    - name: aws_centos
+    - name: aws_defaults
+    - name: aws_flatcar
+    - name: aws_rhel
+    - name: aws_rockylinux
+    - name: azure_default
+    - name: digitalocean_default
+    - name: equinixmetal_default
+    - name: hetzner_default
+    - name: openstack_default
+    - name: vsphere_default
+    - name: vsphere_flatcar
+
+- scenario: upgrade_containerd_external
+  initVersion: v1.21.14
+  upgradedVersion: v1.22.11
+  infrastructures:
+    - name: aws_amzn
+    - name: aws_centos
+    - name: aws_defaults
+    - name: aws_flatcar
+    - name: aws_rhel
+    - name: aws_rockylinux
+    - name: azure_default
+    - name: digitalocean_default
+    - name: equinixmetal_default
+    - name: hetzner_default
+    - name: openstack_default
+    - name: vsphere_default
+    - name: vsphere_flatcar
+
+- scenario: upgrade_containerd_external
+  initVersion: v1.22.11
+  upgradedVersion: v1.23.8
```

Comment:
Why don't we have 1.23 to 1.24 upgrade tests?

Reply:
it was missing for non external containerd too, probably it was too fresh version at a time and later forgotten to be added :shrug: 

will add (+ external conformance too)